README.md

```diff
@@ -204,6 +204,24 @@ Steps for using the configuration:
 6. Now the Flameshot entry should appear in the list. Click _Apply_ to apply the changes.
 7. If you want to change the defaults, you can expand the entry, select the appropriate action and modify it as you wish; the process is pretty mush self-explanatory.
 
+### On Ubuntu (Tested on 18.04)
+
+Taken from [adaptation](https://askubuntu.com/posts/1039949/revisions) of [Pavel Answer on askubuntu](https://askubuntu.com/revisions/1036473/1). To use flameshot instead of default screenshot application in ubuntu we need to release the binding on <kbd>Prt Sc</kbd> key, and then create a new binding for `/usr/bin/flameshot gui`.
+
+1. Release the binding on <kbd>Prt Sc</kbd> using the following command.
+
```

Comment:
the empty lines aren't necessary, please use correct markdown syntax to create newlines (ending a sentence with 2 whitespace chars + newline)

```
this is a sentence followed by 2 whitespace chars->  
this is a new line
```

Reply:
Actually for that part with empty lines I was following the section `On KDE Plasma Desktop`. So that the style is similiar.  So should I just remove the empty lines?  
As for the empty lines between the ordered list I'll remove it ASAP